### ISSUE TITLE
convert keras model to tensorflowjs with weight mismatch, layer with groups=2

### ISSUE THREAD

**Issue by Allenhuang0708** [tensorflow/tfjs#3900]
Hi: 

I am building a keras model,load the pretained weight to it and then try to convert it to tfjs, the problem comes at one layer which set the groups parameter to 2.  I compared the weight shape with the pretranined shape and the keras model, it matches, but when I convert it to tfjs and load it on brower, it gave an error of mismatch.

code: 
eyes_model = keras.Sequential(
    [
        keras.Input(shape=(224, 224, 3)),
        layers.Conv2D(96, kernel_size=11, strides=4, padding='valid', activation='relu'),
        layers.MaxPooling2D(pool_size=3, strides=2),
        layers.Conv2D(256, kernel_size=5, strides=1, padding='same', groups=2, activation='relu'), # This is the layer mismatch happens
        layers.MaxPooling2D(pool_size=3, strides=2),\
        layers.Conv2D(384, kernel_size=3, strides=1, padding='same', activation='relu'),
        layers.Conv2D(64, kernel_size=1, strides=1, padding='valid', activation='relu')  
     ]
)
The right weight should be: [5, 5, 48, 256] which is the shape of the pretrained weight, but when I convert and load it to brower, the errors happans:

Uncaught (in promise) Error: Shape mismatch: [5,5,96,256] vs. [5,5,48,256]
    at checkShapesMatch (webgazer.js:66610)
    at variables_LayerVariable.write (webgazer.js:66578)
    at webgazer.js:66781
    at Array.forEach (<anonymous>)
    at batchSetValue (webgazer.js:66779)
    at models_Sequential.loadWeights (webgazer.js:70579)
    at loadLayersModelFromIOHandler (webgazer.js:73987)

The converting is done by: 
tfjs.converters.save_keras_model(model)

**Comment by rthadur** [comment]
can you please provide tfjs version ?

**Comment by majercakdavid** [comment]
Hi @rthadur  I can observe just the same behavior for the tfjs version 3.3.0. However, I would consider this as a feature request as per API (https://js.tensorflow.org/api/latest/#conv2d) TFJS does not yet support conv2d layers with groups. So even though the information about the groups is embedded into the tfjs model json file its not considered during the parsing.

**Comment by rthadur** [comment]
@majercakdavid thank you , marking this as a feature request.

**Comment by gaikwadrahul8** [comment]
Hi, @Allenhuang0708 

Apologize for the delayed response and It seems like we haven't implemented this feature as of now if I'm not wrong and I have checked `args` of [tf.layers.conv2d](https://js.tensorflow.org/api/4.4.0/#layers.conv2d) and it does not have `groups` arg so Just to confirm, are you still looking for this feature please ? Thank you!
